### ISSUE TITLE
is it possible to get the version number for a beta app?

### ISSUE THREAD

**Issue by shifoc** [facundoolano/google-play-scraper#535]
For example WhatsApp Beta

**Comment by 89z** [comment]
please provide link or app ID

**Comment by shifoc** [comment]
com.whatsapp

**Comment by 89z** [comment]
like this?

~~~
Title: WhatsApp Messenger
Creator: WhatsApp LLC
UploadDate: Apr 18, 2022
VersionString: 2.22.8.79
VersionCode: 220879003
NumDownloads: 7.061 B
Size: 40.121 MB
Files: 1
Offer: 0 USD
~~~

**Comment by shifoc** [comment]
I want the beta version which is 2.22.9.10 now. 2.22.8.78 is the stable version

**Comment by shifoc** [comment]
I want the beta version which is 2.22.9.10 now. 2.22.8.78 is the stable version

**Comment by 89z** [comment]
Is that available on the public website?

**Comment by shifoc** [comment]
I guess you need to login to be able to see it

**Comment by 89z** [comment]
No, I don't think so. Same version is listed, regardless of signin status.

**Comment by 89z** [comment]
> Thanks for your interest in becoming a tester for the WhatsApp Messenger app.
> However, at this time, WhatsApp Messenger app's testing program has reached the maximum number of testers that can participate in it and isn't accepting any more testers.

https://play.google.com/apps/testing/com.whatsapp

**Comment by shifoc** [comment]
I think you are right on the web I didn't see  the beta version, only on the play store. Regarding the link you sent you should refresh multiple times and you will get in. You can also try with com.whatsapp.w4b

**Comment by 89z** [comment]
Sorry I am not messing with this any more. Good luck to you.

**Comment by xhyrom** [comment]
any update about this?

**Comment by facundoolano** [comment]
Issue automatically closed. Please create a new one if the issue persists
